packages/braid-design-system/src/lib/components/private/dataAttribute.docs.tsx

```diff
@@ -0,0 +1,43 @@
+import type { ComponentExample } from 'site/types';
+
+import { Notice, Strong, Text, TextLink } from '../';
+
+type CreateDataAttributeDocs = (
+  params: {
+    code: string;
+  } & (
+    | { supportsNativeSyntax: false }
+    | { supportsNativeSyntax: true; componentName: string }
+  ),
+) => ComponentExample;
+
+export const dataAttributeDocs: CreateDataAttributeDocs = ({
+  code,
+  supportsNativeSyntax,
+  ...restProps
+}) => ({
+  label: 'Data attributes',
+  description: (
+    <>
+      <Text>
+        Braid components are very explicit about the properties they accept,
+        which makes providing arbitrary{' '}
+        <TextLink href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes">
```

Comment:
Is this an old URL? This URL currently 301 redirects to: https://developer.mozilla.org/en-US/docs/Learn_web_development/Howto/Solve_HTML_problems/Use_data_attributes

```suggestion
        <TextLink href="https://developer.mozilla.org/en-US/docs/Learn_web_development/Howto/Solve_HTML_problems/Use_data_attributes">
```

<img width="318" alt="Screenshot 2025-02-04 at 3 34 05 pm" src="https://github.com/user-attachments/assets/9b3dc23d-ef8a-4eeb-9651-8fb39ad1a7f2" />

Reply:
As old as can be. Nice catch!